gradle.properties

```diff
@@ -20,6 +20,6 @@ android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 
-gradleVersion=7.1.0-alpha03
+gradleVersion=4.2.0
```

Comment:
don't push the changes in this file, use it for your local copy only

Reply:
gradle.properties has been put into .gitignore

Reply:
No don't modify .gitignore, just don't add the changes of this file to your commits

Reply:
Ok, I removed the change to .gitignore and reverted the change in the gradle.properties file, then pushed. Then, locally, I changed the version number back to 4.2.0 and in future commits I will make sure not to include this change